LocalMultiplayerAgent/Config/MultiplayerSettings.cs

```diff
@@ -46,8 +46,8 @@ public class MultiplayerSettings
         public ProcessStartParameters ProcessStartParameters { get; set; }
 
         public SessionConfig SessionConfig { get; set; }
-        
-       
+
+        public bool forcePullFromAcrOnLinuxContainersOnWindows { get; set; }
```

Comment:
should we have the variable here or in the ImageDetails class? not sure

Reply:
I'm not sure since we moved the variable to LinuxContainersOnWindowsSample.json. I thought ContainerStartParameters include the parameters which we need to start/run Container, so I put the variable outside of it.

Reply:
yeah, I thought about it since it's only relevant to containers. But that's fine, we can leave it there :)